pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotBrokerHealthCheck.java

```diff
@@ -23,12 +23,14 @@
 import io.swagger.annotations.ApiResponse;
 import io.swagger.annotations.ApiResponses;
 import javax.inject.Inject;
+import javax.inject.Named;
```

Comment:
new to java + injection. what is this for?

Reply:
This is for injecting member variables when register the http server api resources